src/fiatconnect/saga.ts

```diff
@@ -150,12 +150,13 @@ export function* handleSubmitFiatAccount({
     quote.getProviderBaseUrl(),
     quote.getProviderApiKey()
   )
+
   const postFiatAccountResponse: Result<PostFiatAccountResponse, ResponseError> = yield call(
-    [fiatConnectClient, 'addFiatAccount'],
+    { context: fiatConnectClient, fn: fiatConnectClient.addFiatAccount },
```

Comment:
is this change required?

Reply:
no its not, just a different way of using `yield call`